app/Widgets/Hero_Slider_Widget.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+namespace Boilerplate\Theme\Widgets;
+
+use JustCoded\WP\Framework\Page_Builder\v25\Page_Builder_Widget;
+use JustCoded\WP\Framework\Web\View;
+
+/**
+ * Class Hero_Slider_Widget
+ * Custom image wordpress widget based on siteorigin widgets bundle
+ */
+class Hero_Slider_Widget extends Page_Builder_Widget {
+	/**
+	 * Hero_Slider_Widget constructor.
+	 */
+	function __construct() {
+		parent::__construct(
+			'hero-image',
+			__( 'Slider image widget' ),
+			array(
+				'description' => __( 'Slider image widget.' ),
+				'has_preview' => true,
```

Comment:
If you don't have good image for slider - than set false here

Reply:
done